monai/transforms/utility/dictionary.py

```diff
@@ -399,15 +400,34 @@ def __init__(
                 dimension will be squeezed.
             update_meta: if `True`, copy `[key]_meta_dict` for each output and update affine to
                 reflect the cropped image
+            list_output: it `True`, the output will be a list of dictionaries with the same keys as original.
```

Comment:
Hi @drbeh, if `list_output` is `True`, it will return a list of dictionaries in your design. Do we need just one dictionary with orginal key and a list of data?

Reply:
Hi @KumoLiu, we want to have a list of dictionaries all with the same keys, so the transforms are being applied individually on each of them.